ruleset/sca/windows/cis_win10_enterprise.yml

```diff
@@ -5863,157 +6365,178 @@ checks:
       - 'r:HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate -> DeferFeatureUpdates -> 1'
       - 'r:HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate -> DeferFeatureUpdatesPeriodInDays -> n:^(\d+) compare >= 180'
 
   - id: 15393
     title: "Ensure 'Select when Quality Updates are received' is set to 'Enabled: 0 days'"
-    description: "This settings controls when Quality Updates are received."
+    description: "This settings controls when Quality Updates are received. The recommended state for this setting is: Enabled: 0 days. Note: If the \"Allow Diagnostic Data\" (formerly \"Allow Telemetry\") policy is set to 0, this policy will have no effect. Note #2: Starting with Windows 10 R1607, Microsoft introduced a new Windows Update (WU) client behavior called Dual Scan, with an eye to cloud-based update management. In some cases, this Dual Scan feature can interfere with Windows Updates from Windows Server Update Services (WSUS) and/or manual WU updates. If you are using WSUS in your environment, you may need to set the above setting to Not Configured or configure the setting Do not allow update deferral policies to cause scans against Windows Update (added in the Windows 10 Release 1709 Administrative Templates) in order to prevent the Dual Scan feature from interfering. More information on Dual Scan is available at these links: - Demystifying “Dual Scan” - WSUS Product Team Blog - Improving Dual Scan on 1607 - WSUS Product Team Blog"
     rationale: "Quality Updates can contain important bug fixes and/or security patches, and should be installed as soon as possible."
-    remediation: "To establish the recommended configuration via GP, set the following UI path to Enabled:0 days: Computer Configuration\\Policies\\Administrative Templates\\Windows Components\\Windows Update\\Windows Update for Business\\Select when Quality Updates are received"
+    remediation: "To establish the recommended configuration via GP, set the following UI path to Enabled:0 days: Computer Configuration\\Policies\\Administrative Templates\\Windows Components\\Windows Update\\Windows Update for Business\\Select when Quality Updates are received. Note: This Group Policy path does not exist by default. An updated Group Policy template (WindowsUpdate.admx/adml) is required - it is included with the Microsoft Windows 10 Release 1607 & Server 2016 Administrative Templates (or newer)."
     compliance:
       - cis: ["18.9.108.4.3"]
-      - cis_csc: ["3.4", "3.5", "4.5"]
+      - cis_csc: ["7.3"]
       - pci_dss: ["6.2"]
-      - nist_800_53: ["SI.2","SA.11","SI.4"]
+      - nist_800_53: ["SI.2", "SA.11", "SI.4"]
       - gpg_13: ["4.3"]
       - gdpr_IV: ["35.7.d"]
       - hipaa: ["164.312.b"]
-      - tsc: ["A1.2","CC6.8"]
+      - tsc: ["A1.2", "CC6.8"]
     condition: all
     rules:
       - 'r:HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate -> DeferQualityUpdates -> 1'
```

Comment:
Please add:
```suggestion
      - 'r:HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate'
      - 'r:HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate -> DeferQualityUpdates -> 1'
```

Reply:
The default value is
```
Enabled: 0 days. (Install new Quality Updates as soon as they are available.)
```
Due to SCA limitations is not possible to check the existence of those registries (any condition and negate the values) at the same time as ensuring both registry values.

Reply:
Oh, you are correct